src/partials/integrations_details_features.html

```diff
@@ -0,0 +1,72 @@
+<!-- Features -->
+<section id="details" class="">
+  <div
+  class="m-auto max-w-4xl px-4 pt-12 pb-20 flex flex-col gap-12 md:gap-15">
+
+    <!--feature one-->
+    <div class="flex flex-col md:flex-row gap-12 md:gap-15 items-center justify-between">
+      <div class="flex flex-col px-1 md:px-0 max-w-[400px]">
+        <h3 class="text-2xl text-gray-800 font-semibold"> {{feature_01}} </h3>
+        <span class="mt-4 text-lg text-gray-800"> {{{description_01}}}  </span>
+      </div>
+      <img
+      class="max-h-[400px] order-1 md:order-none"
+      src="{{img_features_01}}"
+      alt="{{alt_features_01}}"
+      />
+    </div>
+
+    <!--feature two-->
+    <div class="flex flex-col md:flex-row gap-12 md:gap-15 items-center justify-between">
+      <img
+      class="max-h-[400px] order-1 md:order-none"
+      src="{{img_features_02}}"
+      alt="{{alt_features_02}}"
+      />
+      <div class="flex flex-col px-1 md:px-0 max-w-[400px]">
+        <h3 class="text-2xl text-gray-800 font-semibold"> {{feature_02}} </h3>
+        <span class=" mt-4 text-lg text-gray-800"> {{{description_02}}}  </span>
+      </div>
+
+    </div>
+
+    <!--feature three-->
+    <div class="integrations flex flex-col md:flex-row gap-12 md:gap-15 items-center justify-between">
+      <div class="integrations flex flex-col px-1 md:px-0 max-w-[400px]">
+        <h3 class="integrations text-2xl text-gray-800 font-semibold"> {{feature_03}} </h3>
+        <span class="integrations mt-4 text-lg text-gray-800"> {{{description_03}}}  </span>
+      </div>
+      <img
+      class="integrations max-h-[400px] order-1 md:order-none"
+      src="{{img_features_03}}"
+      alt="{{alt_features_03}}"
+      />
+    </div>
+
+    <!--feature four-->
+    <div class="flex flex-col md:flex-row gap-12 md:gap-15 items-center justify-between">
+      <img
+      class="max-h-[400px] order-1 md:order-none"
+      src="{{img_features_04}}"
+      alt="{{alt_features_04}}"
+      />
+      <div class="flex flex-col px-1 md:px-0 max-w-[400px]">
+        <h3 class="text-2xl text-gray-800 font-semibold"> {{feature_04}} </h3>
+        <span class="mt-4 text-lg text-gray-800"> {{{description_04}}}  </span>
+      </div>
+    </div>
+
+    <!--feature five-->
```

Comment:
The features items only differ in the image+text order right? I will see if we can generate these dynamically from a list of features, so we are not hardcoding to exactly 5, but don't worry about it :)